### ISSUE TITLE
Shiny placeholder not working for few pokemon

### ISSUE THREAD

**Issue by Steedr** [Neovitalism/PixelmonExtension#2]
Specifically for Vivillon, Flabébé, Floette and Florges
I think it's because they don't have a regular shiny name as most Pokemons
Vivillion - "archipelago_shiny"
Flabébé - "redshiny"

**Comment by Steedr** [comment]
Turns out its pixelmon bug
```
public boolean isShiny() {
        return this.isPalette("shiny");
    }
```

**Comment by Neovitalism** [comment]
Can add a clause for these checking if the palette has shiny in the name

**Comment by Neovitalism** [comment]
Fixed in [PixelmonExtension 1.0.1](https://github.com/Neovitalism/PixelmonExtension/releases/tag/1.0.1)
Thank you <3
